### DISCUSSION TITLE
Please restore the Rectangular Array in the Sketcher...

### DISCUSSION THREAD

**Discussion by FredLaCasemate** [FreeCAD/FreeCAD#17710] [Code contribution discussions]
The new Sketcher_translate feature is not at all efficient for creating arrays of objects. There are no more construction lines and constraints to modify the angle and distance between elements a posteriori. We end up with independent elements that no longer have the underlying structure of an array.
The old feature (which still exists but is marked obsolete) was much better designed and I am very disappointed that it is no longer present in version 1.0

**Comment by luzpaz** [comment]
This is not a code contribution. Please open a thread on our forum to get more traction on this. Closing
